python/cuml/cuml/dask/metrics/utils.py

```diff
@@ -1,28 +0,0 @@
-#
-# Copyright (c) 2020-2025, NVIDIA CORPORATION.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-
-import cupy as cp
-
-
-def sorted_unique_labels(*ys):
```

Comment:
It looks like this function is also used in some other places, e.g., python/cuml/cuml/metrics/cluster/utils.py. So we should probably keep it.